.github/workflows/build-mesa-host.yml

```diff
@@ -0,0 +1,164 @@
+name: build-mesa-host
+on:
+  workflow_dispatch:
+    inputs:
+      gitref_to_build:
+        description: "gitref tag or hash to build"
+        default: master
+        required: true
+        type: string
+      target:
+        description: "targets to build"
+        default: all
+        required: true
+        type: choice
+        options:
+          - all
+          - aarch64
+          - x86_64
+  schedule:
+    # The times are specified in UTC
+    - cron: "15 13 * * *"
+
+env:
+  debug: debug
+  ephemeral: ephemeral
+
+jobs:
+  check_build_tools:
+    runs-on: ubuntu-latest
```

Comment:
rather use hard coded version to avoid problems if latest switches to whatever is latest at that moment ?